### PR TITLE
Remove net debug

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
## Initial Checklist

- [x] If part of an EPIC, PR git log contains EPIC number.

## Description (git log)

    net_events: remove current net debugging mechanism

    Preparation for moving away from the TC eBPF program types for ingress
    and egress. The current net_debug is also interefering in the normal
    debug output. A similar (or not) debugging approach will be made if/when
    needed.

Related: #2152

## Type of change

- [ ] Bug fix (non-breaking change fixing an issue, preferable).
- [ ] Quick fix (minor non-breaking change requiring no issue, use with care)
- [x] Code refactor (code improvement and/or code removal)
- [ ] New feature (non-breaking change adding functionality).
- [ ] Breaking change (cause existing functionality not to work as expected).

## How Has This Been Tested?

General CI/CD tests.

## Final Checklist:

Pick "Bug Fix" or "Feature", delete the other and mark appropriate checks.

- [x] I have made corresponding changes to the documentation.
- [x] My code follows the style guidelines (C and Go) of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented all functions/methods created explaining what they do.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix, or feature, is effective.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published before.

## Git Log Checklist:

My commits logs have:

- [x] Subject starts with "subsystem|file: description".
- [x] Do not end the subject line with a period.
- [x] Limit the subject line to 50 characters.
- [x] Separate subject from body with a blank line.
- [x] Use the imperative mood in the subject line.
- [x] Wrap the body at 72 characters.
- [x] Use the body to explain what and why instead of how.

### CODE REVIEW

**Comment by rafaeldtinoco** [comment]
This is mostly "proforma" as there are no changes to code.

File: pkg/ebpf/c/tracee.bpf.c [hunk 1]
```diff
@@ -6328,9 +6248,4 @@ SEC("tc")
 int tc_ingress(struct __sk_buff *skb)
 {
     return tc_probe(skb, true);
-}
-
-char LICENSE[] SEC("license") = "GPL";
-#ifndef CORE
-int KERNEL_VERSION SEC("version") = LINUX_VERSION_CODE;
-#endif
+}
```
Comment (NDStrahilevitz) [inline pkg/ebpf/c/tracee.bpf.c:6251]: nit: The newline is gone

**Review by NDStrahilevitz — APPROVED** [review]
Nice. Left one nit, let's wait for @yanivagman

**Review by yanivagman — APPROVED** [review]
LGTM

File: pkg/ebpf/c/tracee.bpf.c (hunk 1 above)
Comment (rafaeldtinoco) [reply pkg/ebpf/c/tracee.bpf.c:6251]: Yep, if this is something to be enforced we should have in .clang-format file as well… Checking...

Comment (rafaeldtinoco) [reply pkg/ebpf/c/tracee.bpf.c:6251]: There does not seem to be a way to enforce that style, so I'm afraid it is a no-go (to have empty line after last line).

**Pull request merged**